src/features/movie/components/imageContainer/ImageContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { MovieProps } from "../../../../models/movie.model";
+
+import { AiOutlineStar } from "react-icons/ai";
+import imgIron from "../../../../assets/images/iron.jpg";
```

Comment:
Will you be removing this? Avoid committing development time code, or make sure you have a TODO to remind you to remove this in the future.

Reply:
Done